coursebuilder/modules/explorer/_static/components/pages/tools-page.html

```diff
@@ -186,7 +186,7 @@ <h4 class="card-title">2-Step Verification</h4>
                 </div>  
                 <div class="col l4 m6 s12">
                     <div class="card">
-                        <img class="card-img-top" src="/modules/explorer/_static/images/google-adsense.jpg" alt="Google AdSense">
+                        <img class="card-img-top" src="/modules/explorer/_static/images/google-asense.jpg" alt="Google">
```

Comment:
@andela-angene What is going on here?

Reply:
@chegejames ad-blockers will block any image with 'adsense' as part of it's name, this can give the tools page a strange look for users with ad-blockers on.